hugo/content/sponsors/resource-center/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "DORA Sponsor Resource Center"
+date: 2024-04-26T20:58:12-04:00
+draft: false
+bannerTitle: DORA Sponsor Resource Center
+bannerSubtitle: Thank you for sponsoring the 2024 DORA Report
+---
+
+The [2024 DORA survey](https://google.qualtrics.com/jfe/form/SV_8uCHA4aRzcGDjg2?source=sponsor) is now open and we appreciate your efforts to help drive global participation in this year's survey.
+
+## Launch your campaign!
+
```

Comment:
I don't think we need line breaks between headings and paragraphs... this might be the cause of some extra spacing I'm seeing on mobile. Can you remove?

Reply:
Fixed(?) in with 2121850dcec271f6ca27d647529633ea7c2db8b6